sdk/cosmosdb/cosmos/src/client/ChangeFeed/InternalChangeFeedOptions.ts

```diff
@@ -12,5 +14,7 @@ export interface InternalChangeFeedIteratorOptions {
 
   continuationToken?: string;
 
-  startTime?: Date;
+  startTime?: Date | string;
```

Comment:
why did we changed it to string?

Reply:
This is for differentiating b/w startFromNow and startFromTime.

Reply:
updated the code.